### PR TITLE
Added Prime Number in C++

### PR DESCRIPTION
**Pull request by daffadaraz**
since there is no prime number sample for C++

- [x]  I created/added a program for prime-num.cpp in C++
- [x]  I created/updated the language README
  - [x]  I added the sample program name to the README without URL

### CODE REVIEW

File: archive/c/c-plus-plus/README.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ Welcome to Sample Programs in C++!
 - [Hello World in C++](https://therenegadecoder.com/code/hello-world-in-c-plus-plus/)
 - [Insertion Sort in C++](https://github.com/TheRenegadeCoder/sample-programs/issues/1240)
 - [Reverse String in C++](https://github.com/TheRenegadeCoder/sample-programs/issues/419)
+- [Prime Number in C++]()
```
Comment (jrg94) [inline archive/c/c-plus-plus/README.md]:
```suggestion
- Prime Number in C++
```

**Comment by jrg94** [comment]
Looks like this is a failing a couple test cases. Take a look!

File: archive/c/c-plus-plus/prime-number.cpp [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    }
+    string tmp = argv[1];
+    if (argc == 1 || argv[1][0] == '\0' || (atoi(argv[1]) == 0 && strcmp(argv[1], "0") != 0) || atoi(argv[1]) < 0 || tmp.find(".") != string::npos ) {
+        cout<<"Usage: please input a non-negative integer\n";
+    } 
+    else {
+        int input = atoi(argv[1]);
+        for(int i = 2; i < input; ++i){
+            if(input % i == 0){
+                cout<<"Not Prime\n";
```
Comment (jrg94) [inline archive/c/c-plus-plus/prime-number.cpp]:
```suggestion
                cout<<"composite\n";
```

Comment (daffadaraz) [reply archive/c/c-plus-plus/prime-number.cpp]: Oh right, i forgot

**Pull request merged**